src/helper/units.ts

```diff
@@ -14,8 +14,11 @@ const arrUnitNames = [
   'Tril',
 ];
 
-export const getUnitNames = (defaultName: string) => {
+export const setDefaultUnitName = (defaultName: string) => {
```

Comment:
The token unit will render as `Unit` instead of the proper token symbol like `PLM`, `PLD`, or `SDN` when the amount is `default` https://github.com/PlasmNetwork/plasm-apps-vue/blob/175de2087fbde506947aaedf44e18b6448bbcd25/src/helper/units.ts#L10
![bandicam 2021-07-12 11-14-36-198](https://user-images.githubusercontent.com/40356749/125246132-6f333a00-e302-11eb-95e0-0ac86c9610b6.jpg)

Reply:
Fix it. should be working now.

Reply:
thank you for the fix. I confirmed that it works!